aptos-move/framework/aptos-framework/sources/aptos_governance.move

```diff
@@ -362,6 +378,15 @@ module aptos_framework::aptos_governance {
         get_signer(signer_address)
     }
 
+    /// Resolve a successful multi-step proposal. This would fail if the proposal is not successful.
+    public fun resolve_multi_step_proposal(proposal_id: u64, signer_address: address, next_execution_hash: vector<u8>): signer acquires GovernanceResponsbility, ApprovedExecutionHashes {
+        voting::resolve_single_step_or_multi_step_proposal<GovernanceProposal>(@aptos_framework, proposal_id, next_execution_hash);
```

Comment:
Don't you also need to add the hash of the next step? Otherwise, it might get rejected by mempool if exceeding the transaction size limit (64 KB)

Reply:
yeah good point - just updated : )